ml-agents/mlagents/trainers/common/nn_policy.py

```diff
@@ -0,0 +1,381 @@
+import logging
+import numpy as np
+from typing import Any, Dict, Optional, List
+
+from mlagents.tf_utils import tf
+
+from mlagents_envs.timers import timed
+from mlagents_envs.base_env import BatchedStepResult
+from mlagents.trainers.brain import BrainParameters
+from mlagents.trainers.models import EncoderType
+from mlagents.trainers.models import LearningModel
+from mlagents.trainers.tf_policy import TFPolicy
+
+logger = logging.getLogger("mlagents.trainers")
+
+LOG_STD_MAX = 2
```

Comment:
Should these be model parameters? Same defaults are OK

Reply:
I made them member variables so we could make them params in the future. But there isn't a good reason for the user to be twiddling these for 99% of the use cases, so I'm not sure about exposing them.